src/ComputeSharp.D2D1/Shaders/Translation/D3DCompiler.cs

```diff
@@ -20,7 +20,7 @@ namespace ComputeSharp.D2D1.Shaders.Translation;
 /// <summary>
 /// A <see langword="class"/> that uses the FXC APIs to compile D2D1 pixel shaders.
```

Comment:
If this is now `D3DCompiler`, should the summary still just say it's for D2D1 ?

Reply:
This is an internal type, so it doesn't really matter. That said it's still just for D2D1, as the APIs it exposes only allow to specifically compile D2D1 shaders and not anything else, as many of the values are hardcoded to suit the needs of the library. I just renamed it `D3DCompiler` for clarity and to better map to the underlying API being used 🙂